R/format_MTA.R

```diff
@@ -0,0 +1,371 @@
+#' Construct standard format data from MTA-PE Evolutionary Ecology Group, University of Pannonia, Hungary (MTA)
+#'
+#' A pipeline to produce the standard format for bird study population
+#' at the MTA-PE Evolutionary Ecology Group, Hungary, administered by
+#' Gabor Seres & András Liker.
+#' The data include 5 sites: Veszprém, Balatonfüred, Szentgál, Vilma-puszta and Gulya-domb.
+#'
+#' This section provides details on data management choices that are unique to
+#' this data. For a general description of the standard format please see see
+#'\href{https://github.com/SPI-Birds/documentation/blob/master/standard_protocol/SPI_Birds_Protocol_v1.0.0.pdf}{here}.
+#'
+#' \strong{xxx}:
+#'
+#' \strong{Latitude, Longitude} The exact coordinates of the nestboxes are not available.
+#' Data owner provided an map of the location, which can be georeferenced in the case of
+#' interest or necessity of the data user. Currently, only general coordinates
+#' for the location are provided.
```

Comment:
Is this still true?

Reply:
Nope, my mistake. The true is that we do not have coordinates for nestboxes, neither we do have any map. I will ask data owners about that.